api/rest/restcore/AuthMiddleware.php

```diff
@@ -51,14 +51,27 @@ public function __invoke( \Slim\Http\Request $request, \Slim\Http\Response $resp
 			}
 		} else {
 			# TODO: add an index on the token hash for the method below
-			$t_user_id = api_token_get_user( $t_authorization_header );
+
+			# Manage multiple authorization header (ex: Basic + token)
+			$authStringArray = explode(', ', $t_authorization_header);
+			$t_user_id = "";
+			$real_auth_header = "";
+			#Search for the token among the different authStrings 
```

Comment:
Replace `#Search for the token among the different authStrings ` with `# Search for the token among the different authorization headers`.  Note the space and changing of authStrings string.

Reply:
Change done